crates/testing/kats/Readme.md

```diff
@@ -0,0 +1,11 @@
+# KATs
+
+This crate provides KAT test vectors for:
```

Comment:
This should mention somewhere that it's internal to libcrux and people should only use at their own risk.